DylibBinder/DBAssociatedType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DylibBinder {
+	// TODO Add AssociatedTypes
+	public class DBAssociatedType {
+		public DBAssociatedType (string name)
+		{
+			Name = name;
+		}
+
+		public string Name { get; }
+	}
```

Comment:
What value is this class adding?

Reply:
Nothing.. yet! This is holding the place value for the associatedTypes that I hope I can capture later